src/openloco/viewportmgr.cpp

```diff
@@ -16,4 +16,16 @@ namespace openloco::ui::viewportmgr
         auto arr = (std::array<viewport*, max_viewports>*)_viewports.get();
         return *arr;
     }
+
+    void create(window* window, int16_t x, int16_t y, uint16_t width, uint16_t height)
+    {
+        registers regs;
+        regs.edx = 0x17FF17FF;
```

Comment:
edx is either the `thing_id` or the x, y tile coordinates of the viewport. There is also flags encoded into bit 31 and bit 30. Bit 31 is set when a `thing_id` is being passed. bit 30 does something im not entirely sure to do with the zoom.